lib/3rd_party/nanopb/CMakeLists.txt

```diff
@@ -8,7 +8,34 @@ target_sources(nanopb
         nanopb/pb_encode.c
 )
 
+# copy public header files for correct include prefix
+
+file(COPY
+        nanopb/pb.h
+        nanopb/pb_encode.h
+        nanopb/pb_decode.h
+    DESTINATION
+        ${THIRDPARTY_INCLUDE_TMP_DIR}/nanopb
+)
+
+
 target_include_directories(nanopb
     PUBLIC
-        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
+        $<BUILD_INTERFACE:${EVEREST_CORE_LIB_INCLUDE_DIR}>
+)
+
+set_target_properties(nanopb PROPERTIES OUTPUT_NAME "everest_nanopb")
+
+install(TARGETS nanopb
+    EXPORT everest-core-targets
+    LIBRARY
+)
+
+# FIXME (aw): this would collide with the real nanopb library if
```

Comment:
Does this comment still apply? This will (only) collide if nanopb is installed into /<libdir>/3rd_party doesn't it? If it's still a problem why not fix it in this PR?

Reply:
:+1: will remove it